src/number_theory/divisors.lean

```diff
@@ -398,4 +400,77 @@ begin
   exact nat.lt_succ_of_le (nat.divisor_le (nat.mem_divisors.2 ⟨ha, h⟩))
 end
 
+
+
+/-- # Prime divisors -/
+
+lemma dvd_of_mem_factors {n p : ℕ} (h : p ∈ n.factors) : p ∣ n
+  :=
+begin
+  by_cases hn : n = 0,
+  { convert dvd_zero p },
+  { rwa ←mem_factors_iff_dvd (pos_iff_ne_zero.mpr hn) (prime_of_mem_factors h) }
+end
```

Comment:
This lemma should go right next to `nat.mem_factors_iff_dvd` in `data/nat/prime.lean`